### DISCUSSION TITLE
Alarms & Conditions and Alarm History supported by asyncua?

### DISCUSSION THREAD

**Discussion by nkawerau** [FreeOpcUa/opcua-asyncio#1182] [Q&A]
Hello,

I would like to know if asyncua supports on the client side Alarms & Conditions and Alarm History as it is described here:
https://reference.opcfoundation.org/Core/Part9/v105/docs/

If yes, can you say something about the level of maturity? I would like to use it in combination with an OPC-UA-Server provided by the OEM (Bachmann) of my PLC hardware. If anyone happens to have any experience with this.

**Comment by AndreasHeine** [comment]
https://github.com/FreeOpcUa/opcua-asyncio/blob/master/examples/client_to_uanet_alarm_conditions.py
